client/executor/Cargo.toml

```diff
@@ -47,7 +47,6 @@ sp-state-machine = { version = "0.10.0-dev", path = "../../primitives/state-mach
 sp-runtime = { version = "4.0.0-dev", path = "../../primitives/runtime" }
 sp-tracing = { version = "4.0.0-dev", path = "../../primitives/tracing" }
 sp-maybe-compressed-blob = { version = "4.0.0-dev", path = "../../primitives/maybe-compressed-blob" }
-sp-tracing = { version = "3.0.0", path = "../../primitives/tracing" }
 tracing = "0.1.25"
```

Comment:
Why we don't need it?

Reply:
Are you asking about `tracing = "0.1.25"`? It's not removed